api/BackgroundFetchRegistration.json

```
@@ -391,10 +391,11 @@
           }
         }
       },
-      "onprogress": {
+      "progress_event": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/BackgroundFetchRegistration/onprogress",
-          "spec_url": "https://wicg.github.io/background-fetch/#dom-backgroundfetchregistration-onprogress",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/BackgroundFetchRegistration/progress_event",
+          "description": "<code>progress</code> event",
+          "spec_url": "https://wicg.github.io/background-fetch/#eventdef-backgroundfetchregistration-progress",
```

Seems appropriate to link to the whole section in the spec since this BCD entry (and the MDN page) are about both onprogress and progress event now.

```suggestion
          "spec_url": "https://wicg.github.io/background-fetch/#background-fetch-registration-events",
```

---

I hesitated. I will fix this.